pyramid_oereb/views/webservice.py

```diff
@@ -68,7 +68,14 @@ def get_versions(self):
                 ]
             }
         }
-        renderer_name = 'json' if self._is_json() else 'pyramid_oereb_versions_xml'
+        # Try - catch for backward compatibility with old specification.
+        try:
+            format = self.__validate_format_param__(['xml', 'json'])
```

Comment:
Can you rename this variable? format seems to be a function of the Python standard library

Reply:
Codacy complained about this.

Reply:
Oh, okay, I'll do that.Thanks